osconfig_tests/test_suites/package_management/package_management.go

```diff
@@ -354,11 +354,11 @@ func runPackageInstallFromNewRepoTest(ctx context.Context, testCase *junitxml.Te
 func packageManagementTestCase(ctx context.Context, testSetup *packageManagementTestSetup, tests chan *junitxml.TestCase, wg *sync.WaitGroup, logger *log.Logger, regex *regexp.Regexp, testProjectConfig *testconfig.Project) {
 	defer wg.Done()
 
-	createOsConfigTest := junitxml.NewTestCase(testSuiteName, fmt.Sprintf("[%s/CreateOsConfig] Create OsConfig", testSetup.image))
-	packageInstallTest := junitxml.NewTestCase(testSuiteName, fmt.Sprintf("[%s/PackageInstall] Package installation", testSetup.image))
-	packageRemovalTest := junitxml.NewTestCase(testSuiteName, fmt.Sprintf("[%s/PackageRemoval] Package removal", testSetup.image))
-	packageInstallRemovalTest := junitxml.NewTestCase(testSuiteName, fmt.Sprintf("[%s/PackageInstallRemoval] Package no change", testSetup.image))
-	packageInstallFromNewRepoTest := junitxml.NewTestCase(testSuiteName, fmt.Sprintf("[%s/PackageInstallFromNewRepo] Add a new package from new repository", testSetup.image))
+	createOsConfigTest := junitxml.NewTestCase(testSuiteName, fmt.Sprintf("[createosconfig] [%s] Create OsConfig", testSetup.image))
```

Comment:
This seems redundant and doesn't match what we have elsewhere, do you plan to update the other tests as well?
Also we should stick with CamelCase
[PackageManagementTests] [createosconfig] [projects/centos-cloud/global/images/family/centos-6] Create OsConfig
[InventoryTests] [projects/ubuntu-os-cloud/global/images/family/ubuntu-1804-lts] Check Hostname

Reply:
the format that we decided on "[test-suite] [testcase] [Image name] description". i am planning to update to other test suites as well. actually let me do that in this pr itself.
CamelCase makes sense. will change that as well

Reply:
changed the test name for every test suite. also change some logs to remove redundant info in log lines